numba/core/utils.py

```diff
@@ -396,7 +418,7 @@ def __len__(self):
         return len(self._values)
 
     def __iter__(self):
-        return iter(k for k in sorted(self._values))
+        return iter(k for k in (self._values))
```

Comment:
```suggestion
        return iter(k for k in sorted(self._values))
```
Shouldn't have removed the `sorted`. It was part of debugging and i  accidentally added to the patch. It turns out only the RVSDG frontend uses this so no tests were affected.